src/components/form-events/mock/events.ts

```diff
@@ -0,0 +1,231 @@
+            };
+            name?: string;
+            iconUri?: string;
+        };
+        conferenceId?: string;
+    };
+    reminders?: {
+        useDefault?: boolean;
+    };
+    eventType?: string;
+};
+export const events: CalendarEvent[] = [
+    {
+        kind: 'calendar#event',
+        etag: '"3429169713576000"',
+        id: '7rr4p6c3793glnq9bghg5nom3d',
+        status: 'confirmed',
+        htmlLink:
+            'https://www.google.com/calendar/event?eid=N3JyNHA2YzM3OTNnbG5xOWJnaGc1bm9tM2QgdDI0MTQ5ODY1QG0',
+        created: '2024-05-01T17:34:16.000Z',
+        updated: '2024-05-01T17:34:16.788Z',
+        summary: 'test event ',
+        creator: {
+            email: 't24149865@gmail.com',
+            self: true,
+        },
+        organizer: {
+            email: 't24149865@gmail.com',
+            self: true,
+        },
+        start: {
+            dateTime: '2024-05-06T09:00:00+03:00',
+            timeZone: 'Europe/Moscow',
+        },
+        end: {
+            dateTime: '2024-05-06T11:00:00+03:00',
+            timeZone: 'Europe/Moscow',
+        },
+        iCalUID: '7rr4p6c3793glnq9bghg5nom3d@google.com',
+        sequence: 0,
+        reminders: {
+            useDefault: true,
+        },
+        eventType: 'default',
+    },
+    {
+        kind: 'calendar#event',
+        etag: '"3429169724960000"',
+        id: '29sd2c7udn33r9n6k3718kk3cm',
+        status: 'confirmed',
+        htmlLink:
+            'https://www.google.com/calendar/event?eid=MjlzZDJjN3VkbjMzcjluNmszNzE4a2szY20gdDI0MTQ5ODY1QG0',
+        created: '2024-05-01T17:34:22.000Z',
+        updated: '2024-05-01T17:34:22.480Z',
+        summary: 'test event',
+        creator: {
+            email: 't24149865@gmail.com',
+            self: true,
+        },
+        organizer: {
+            email: 't24149865@gmail.com',
+            self: true,
+        },
+        start: {
+            dateTime: '2024-05-08T10:45:00+03:00',
+            timeZone: 'Europe/Moscow',
+        },
+        end: {
+            dateTime: '2024-05-08T13:00:00+03:00',
+            timeZone: 'Europe/Moscow',
+        },
+        iCalUID: '29sd2c7udn33r9n6k3718kk3cm@google.com',
+        sequence: 0,
+        reminders: {
+            useDefault: true,
+        },
+        eventType: 'default',
+    },
+    {
+        kind: 'calendar#event',
+        etag: '"3429173227184000"',
+        id: '3tg8s7ku52po499o1puvtif95f',
+        status: 'confirmed',
+        htmlLink:
+            'https://www.google.com/calendar/event?eid=M3RnOHM3a3U1MnBvNDk5bzFwdXZ0aWY5NWYgdDI0MTQ5ODY1QG0',
+        created: '2024-05-01T17:16:30.000Z',
+        updated: '2024-05-01T18:03:33.592Z',
+        summary: 'test Event',
+        description:
+            '<br><br><span>Lorem ipsum dolor sit amet consectetur adipisicing elit. Nisi aut tempora quod, possimus recusandae quia et sit perferendis, aliquam vel harum voluptatem fuga nesciunt. Harum, beatae expedita! Animi, ullam culpa.</span>',
```

Comment:
hahahaha br br. 

Well for the mock it is fine, but you should not count that we will have it from api